fuzzers/int_generate.py

```diff
@@ -54,10 +54,11 @@
         if tile not in tiledata:
             tiledata[tile] = {"pips": set(), "srcs": set(), "dsts": set()}
 
-        if pip in pipdata:
-            assert pipdata[pip] == (src, dst)
+        tile_lr = ("_".join(tile.split("_")[0:2]))
```

Comment:
Not all interconnect fuzzers have _LR suffixes.  This may break in those cases.

Reply:
@litghost The CI results and diffs look fine to me? Could you take a look and tell me if you see something plain wrong there?

Reply:
I'd like to hold off on this PR until 044 is enabled.